.github/workflows/CI.yml

```
@@ -55,7 +55,7 @@ jobs:
           version: '1'
       - uses: julia-actions/julia-buildpkg@v1
       - run: julia -e 'using Pkg; Pkg.add("Aqua")'
```

```suggestion
      - run: julia --color=yes -e 'using Pkg; Pkg.add("Aqua")'
```
although I feel like this should be part of the tests, instead of something to run separately.